Frontend/ui-customer/src/Screens/CustomerUtils/MyRidesScreen/Controller.purs

```diff
@@ -321,7 +325,7 @@ myRideListTransformer state listRes = filter (\item -> (any (_ == item.status) [
   , showDestination : if (decodeAddress $ Booking destination) == "" then "gone" else "visible" 
   , rideScheduledTime : fromMaybe "" ride.rideScheduledTime
   , totalTime : show (runFn2 differenceBetweenTwoUTCInMinutes endTime startTime) <> " min"
-  , vehicleModel : rideDetails.vehicleModel
+  , vehicleModel : if (rideDetails.vehicleModel `DA.elem` ["", "Unkwown"]) then fromMaybe (HU.getVariantRideType rideDetails.vehicleVariant) ride.serviceTierName else rideDetails.vehicleModel
```

Comment:
can we make a type for this instead of using string here, It was unkown at one place and here it is Unkwown .

Reply:
Fixed here, can we keep it `"Unkown"` for now for proper visibility for frontend aligning with backend. If backend corrects this spelling then we can also update similarly (just replace Unkown with Unknown)